include/dsn/cpp/auto_codes.h

```diff
@@ -167,56 +167,6 @@ namespace dsn
 #endif
     };
 
-    class atom_int
```

Comment:
The atomic int value in "app_state" is removed from the thrift IDL file. I think we shouldn't transfer atomic value through network, as in most cases atomic value are used for internal states.